gkeys/gkeys/action_map.py

```diff
@@ -174,6 +174,15 @@
  Gkey task results:
     Done.''',
          }),
+    ('send-key', {
+        'func': 'sendkey',
+        'options': ['category', 'nick', 'name', 'fingerprint', 'keyid', 'keys',
+            'keydir', 'keyring'],
+        'desc': '''Uploads the selected gpg key''',
+        'long_desc': '''Uploads the selected gpg key''',
+        'example': '''gkeys send-key -C gentoo -n gkeys''',
```

Comment:
While this is fine, it would be better to use an example a normal user might do like send his personal or new gkeys-gen created key.